ui/pages/confirm-decrypt-message/confirm-decrypt-message.component.js

```diff
@@ -19,6 +19,7 @@ import { formatCurrency } from '../../helpers/utils/confirm-tx.util';
 import { getValueFromWeiHex } from '../../../shared/modules/conversion.utils';
 import { COPY_OPTIONS } from '../../../shared/constants/copy';
 
+// As of this PR; This component is is only used in a test file
```

Comment:
```suggestion
// As of this [PR](https://github.com/MetaMask/metamask-extension/pull/27097); This component is is only used in a test file
```

Reply:
Oh, I thought i removed that comment, removing now; component is rendered when decrypting msgs